apps/api/src/routers/admin.py

```diff
@@ -210,6 +210,61 @@ async def confirm_attendance() -> None:
         )
 
 
+@router.get(
+    "/attending",
+    dependencies=[
+        Depends(require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER]))
+    ],
+)
+async def attending() -> list[dict[str, object]]:
+    """Fetch all applicants who have a status of ATTENDING"""
+    records = await mongodb_handler.retrieve(
+        Collection.USERS,
+        {"role": Role.APPLICANT, "status": Status.ATTENDING},
+        [
+            "_id",
+            "status",
+            "role",
+            "application_data.first_name",
+            "application_data.last_name",
+        ],
+    )
+
+    return records
+
+
+@router.post("/checkin/{uid}")
+async def checkin(
+    uid: str,
+    associate: User = Depends(
+        require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER])
+    ),
+) -> None:
+    """Check-in applicant at IrvineHacks"""
+    record: Optional[dict[str, object]] = await mongodb_handler.retrieve_one(
+        Collection.USERS, {"_id": uid, "role": Role.APPLICANT}
+    )
+    if not record:
+        raise HTTPException(status.HTTP_404_NOT_FOUND)
+    elif record["status"] != Status.ATTENDING:
+        raise RuntimeError("Applicant status must be ATTENDING")
```

Comment:
Issue: Instead of raising a runtime error, a `403 Forbidden` status should be returned (`HTTPException`).